mockOSApi.v2/Data/DTO/ProcessDto.cs

```diff
@@ -0,0 +1,78 @@
+using mockOSApi.Models;
+using mockOSApi.Repository;
+
+namespace mockOSApi.DTO;
+
+public class PriorityClass()
```

Comment:
So this class aggregates a priority to a Process?
I would suggest you just include the priority inside the process.
Is there another reason why you made it separately?

Reply:
removed this, will refactor